.github/workflows/tox.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
```

Comment:
Looks like two spaces has crept in here?  Is this a new standard?

Reply:
Thank you for the review @ajkavanagh , much appreciated!

I don't see the extra space though? I did convert all the integers to string versions for consistency though, as a bare `3.10` will be interpreted as `3.1`.

I wonder if GitHub's representation of that change poses as an optical illusion wrt. the number of spaces? :-)

Reply:
> I wonder if GitHub's representation of that change poses as an optical illusion wrt. the number of spaces? :-)

I think that must be it!  I even pulled down the PR and checked.  How strange.  it must be the `', '` combination, highlighted in green, vs red for the previous version, that makes it look wider.